### DISCUSSION TITLE
cool script but there is no main function of user selection by country, region, city.

### DISCUSSION THREAD

**Discussion by excalibur-web** [pH7Software/pH7-Social-Dating-CMS#837] [Ideas/Suggestions]
Hello Pierre-Henry!

**Comment by excalibur-web** [comment]
need to use ajax autocomplete

**Comment by polynamaude** [comment]
@excalibur-web Go use Una (https://www.una.io) or OpenHUB

**Reply by polynamaude** [reply]
@excalibur-web You are a real punk !
You make me laugh...
If you need a job, go on Upwork.
![Screenshot from 2020-11-18 00-48-01](https://user-images.githubusercontent.com/21186871/99490398-c03b8b80-2937-11eb-86a3-3183ebc9c4db.png)

You shall learn the basis regarding the rules of the medium you are using. As a Noobs on Github, I'll teach you that it's more than rude to go ask for money on the support page of software.

Also, it's not a limitation but a bug, like there's many other, for example, when you follow someone, you see that they accepted you even if it's not the case.

And you 100$ just make you look foolish. That's close to half hour of work of a real programmer, in my country.

**Reply by excalibur-web** [reply]
Ok, I understood everything

**Comment by polynamaude** [comment]
@excalibur-web You think we don't know that it needs AJAX auto-complete. There's many other bugs, including one that show up when you reach 2000 users. And this hasn't got fixed and the author of this software don't seem to have interest in fixing it either.

So don't waste your time.

It's better to start from zero than try to fix this.

**Comment by polynamaude** [comment]
@excalibur-web Look at all the other issues...
